openvdb_ax/openvdb_ax/compiler/Compiler.cc

```diff
@@ -682,6 +686,12 @@ Compiler::compile(const ast::Tree& tree,
         CustomData::Ptr data,
         Logger& logger)
 {
+    // @todo  Not technically necessary for volumes but does the
+    //   executer/bindings handle this?
+    if (!verifyTypedAccesses(tree, logger)) {
+        return nullptr;
+    }
```

Comment:
pretty sure the bindings does not as it doesn't care about type, only name.

Reply:
I don't think I changed the behaviour of this here, not sure what you mean?

Reply:
this is a comment on the comment

Reply:
As commented elsewhere, do we want to terminate compilation here? As this would still get through generate() even if the typed accesses are wrong no?

Reply:
I think that this last comment still stands. Even if the typed accesses are wrong (and so should log errors) I dont think it _needs_ to early exit from compilation here. Is it useful to continue compilation and log all subsequent errors that might occur due to these wrong type accesses?